test/framework/deployments/testserver/kubernetes.go

```diff
@@ -311,17 +312,21 @@ func (k *k8SDeployment) Delete(cluster framework.Cluster) error {
 	// This means that namespace is no longer available so the code below would throw an error
 	// If we ever switch DemoClient to be deployment and remove manual deletion of TestNamespace
 	// then we can rely on code below to delete tht deployment.
-
-	// k8s.KubectlDeleteFromString(
-	// 	cluster.GetTesting(),
-	// 	cluster.GetKubectlOptions(framework.TestNamespace),
-	// 	service,
-	// )
-	// k8s.KubectlDeleteFromString(
-	// 	cluster.GetTesting(),
-	// 	cluster.GetKubectlOptions(framework.TestNamespace),
-	// 	fmt.Sprintf(deployment, k.opts.Mesh, framework.GetUniversalImage()),
-	// )
+	resourcePattern := `
+apiVersion: %s
+kind: %s
+metadata:
+  name: %s`
+	k8s.KubectlDeleteFromString(
+		cluster.GetTesting(),
+		cluster.GetKubectlOptions(k.opts.Namespace),
+		fmt.Sprintf(resourcePattern, "v1", "Service", k.opts.Name),
+	)
+	k8s.KubectlDeleteFromString(
+		cluster.GetTesting(),
+		cluster.GetKubectlOptions(k.opts.Namespace),
+		fmt.Sprintf(resourcePattern, "apps/v1", "Deployment", k.opts.Name),
+	)
```

Comment:
feel free to ignore it but nit: instead of building a resource you can just do
```
k8s.RunKubectlE(c.GetTesting(), c.GetKubectlOptions(k.opts.Namespace), "delete", "service", k.opts.Name)
```